### DISCUSSION TITLE
Use of the NoIP service

### DISCUSSION THREAD

**Discussion by Ramses7420** [teslamate-org/teslamate#3715] [Q&A]
I use the NoIP service and therefore I have a redirection address. When I connect to my teslamate, I access Teslamate's PGAE of Startrae, but when I want to see a section I have an error that the requested page does not exist. How to do ?
THANKS

![SCR-20240302-enc](https://github.com/teslamate-org/teslamate/assets/54696668/87237724-12dd-4c66-b676-c6702a87269b)

**Comment by dyxyl** [comment]
Maybe you need to change the URLs on the Settings page?

**Comment by Ramses7420** [comment]
If I change the addresses in the settings, it does not accept

**Comment by dyxyl** [comment]
I don't know what you mean by "it does not accept".
